subgraphs/synapse/README.md

```diff
@@ -0,0 +1,105 @@
+# Synapse 
+
+Synapse is a cross-chain liquidity network secured by validators utilizing multi-party computation (MPC) and threshold signature schemes (TSS). It combines AMMs from multiple chains with its own liquidity pools to allow efficient and low slippage transfer of assets between chains. All in all, Synapse allows users to effortlessly and seamlessly bridge or swap assets between chains, with over $10.5 billion USD in bridged volume. 
+
+## Usage Metrics
+
+`Active Users`, `Total Unique Users` & `Daily Transaction Count`
+
+Users of Synapse are those taking the following actions:
+
+Bridge Assets
+Swap Assets
+Liquidity Pools
+Deposit Assets
+Withdraw Assets
+Stake LP position
+
```

Comment:
Can also send messages through the bridge (currently only on certain chains)

https://github.com/synapsecns/synapse-contracts/commit/f405d0b0e38fabd1048a7a32ad53ec966e4957fb#diff-c6595f0d5324e1295270d210b5ea71c8005443851b8bb92f180ad5c4780ddaba

Reply:
added